composer.json

```diff
@@ -19,6 +19,7 @@
         }
     ],
     "require": {
+        "php": ">=7.1",
         "silverstripe/framework": "^4.5",
```

Comment:
The `silverstripe/framework ^4.5` dependency here already covers this PHP requirement, so this isn't necessary. If you'd like to change the way we handle this, maybe put together an RFC.

Reply:
We seems to be flip-floping on this.  Previously there had been some agreement that we'd put this requirement in where needed.  There has been at least one instance where the 'framework is always included' turned out to be a false assumption and it caused a travis failure.

Reply:
My recollection of those discussions was that we'd add it directly for non-core modules, and for core modules, we'd keep it centralised in `framework` and make sure version requirements were kept in sync between core modules to avoid the issue we faced with the CMS module happening again.

Reply:
Have removed php >=7.1